rdt-open-reader/src/covidIndonesia/assets/ec_client_fields.json

```diff
@@ -61,6 +61,13 @@
             "field": "obs.fieldCode",
             "concept": "patient_occupation"
           }
+        },
+        {
+          "column_name": "last_interacted_with",
```

Comment:
cool, are you sure this is the correct field and not event date? are they the same?

Reply:
yes, this is the timestamp field and equivalent to `eventDate` column in event table and `updatedAt` column in client table.